dev/build/windows/makecoq_mingw.sh

```diff
@@ -794,10 +794,10 @@ function make_ocaml {
     # TODO: this might not work if PREFIX contains spaces
     sed -i "s|^PREFIX=.*|PREFIX=$PREFIXOCAML|" config/Makefile
     
-    # We don't want to mess up Coq's dirctory structure so put the OCaml library in a separate folder
-    # If we refer to the make variable ${PREFIX} below, camlp4 ends up having a wrong path:
+    # We don't want to mess up Coq's directory structure so put the OCaml library in a separate folder
+    # If we refer to the make variable ${PREFIX} below, camlp5 ends up having the wrong path:
     # D:\bin\coq64_buildtest_abs_ocaml4\bin>ocamlc -where => D:/bin/coq64_buildtest_abs_ocaml4/libocaml
-    # D:\bin\coq64_buildtest_abs_ocaml4\bin>camlp4 -where => ${PREFIX}/libocaml\camlp4
+    # D:\bin\coq64_buildtest_abs_ocaml4\bin>camlp5 -where => ${PREFIX}/libocaml\camlp5
```

Comment:
I don't know whether the change `camlp4` -> `camlp5` in these comments is relevant or not.

Reply:
I will revert this change.  These look specific to a single developer's environment.  \"coq64_buildtest_abs_ocaml4" doesn't appear anywhere else in the code base.  Not every Windows system has a D: drive (\"D:\bin\...").  Also \"coq64_buildtest_abs_ocaml**4**\bin>camlp**5**" looks pretty odd.